src/daria/corrections/shape/curvature.py

```diff
@@ -116,6 +119,11 @@ def __init__(
 
         # Initialize cache for precomputed transformed coordinates
         self.cache = {}
+        self.use_cache = self.config.get("use_cache", False)
```

Comment:
I guess that you have tested this, but what is happening here? it looks to me like you would need to get use_cache via the config, although you suggest above that it could be fed through the kwargs?

Reply:
You are right. Thanks!